templates/navigation.html

```diff
@@ -194,24 +194,43 @@ <h3 class="title">${_("Please use Firefox or Safari")}</h3>
 </div>
 <%block name="js_extra">
 <script type="text/javascript">
-  <!--
-  var unityObjectUrl = "http://webplayer.unity3d.com/download_webplayer-3.x/3.0/uo/UnityObject2.js";
-  if (document.location.protocol == 'https:')
-     unityObjectUrl = unityObjectUrl.replace("http://", "https://ssl-");
-  document.write('<script type="text\/javascript" src="' + unityObjectUrl + '"><\/script>');
-  -->
-</script>
-<script type="text/javascript">
 ;(function($){
-  var unity = new UnityObject2(),
-      $warning = $('#unity-browser-support');
+  var $warning = $('#unity-browser-support');
+
+  /**
+   * Simple version of Unity's browser check logic.
+   * @return {Boolean}
+   */
+  var isSafari = function() {
+    var ua = navigator.userAgent;
+
+    if (ua.indexOf("Opera") != -1 || ua.indexOf("Chrome") != -1) {
+        return false;
+    } else if (ua.indexOf("Safari") != -1) {
```

Comment:
Can we make it more clear: a) have list of supported browsers, b) have list of unsuported, and check indexes of the list elements?